docs/modules/ROOT/pages/runbooks/CephDeviceFailurePredicted.adoc

```diff
@@ -0,0 +1,41 @@
+= Alert rule: CephDeviceFailurePredicted
```

Comment:
We can probably disable all `CephDevice` alerts by default, as we tend to run Ceph on virtual disks for which there's no real useful S.M.A.R.T. info available.